.github/workflows/lint-eslint.yml

```diff
@@ -22,17 +33,17 @@ jobs:
 
     steps:
       - name: Checkout
-        uses: actions/checkout@v3
+        uses: actions/checkout@755da8c3cf115ac066823e79a1e1788f8940201b # v3
```

Comment:
This hash is from version 3.2.0

https://github.com/actions/checkout/commit/755da8c3cf115ac066823e79a1e1788f8940201b

Why did you replaced the latest version 3 by a commit hash?

Reply:
Ref is https://github.com/nextcloud/.github/pull/101

I just copied.